accounts/views.py

```diff
@@ -57,9 +70,10 @@ def signup(request):
         if form.is_valid():
             user = form.save()
             login(request, user)
-            # request.session["onboarding"] = True TODO link to onboarding later
-            # return render(request, "accounts/onboarding.html")
-            return redirect("landing_page:index")
+            request.session["onboarding"] = (
```

Comment:
i feel like this... also shouldn't be three lines?
like this is an appropriate above code comment

Reply:
Yeah for some reason it made the inline comment into that so I just left it that way